docs/Agent/config/config.md

```diff
@@ -0,0 +1,22 @@
+### Configuration
+
+**Note**: This Documentation is only required for administrative purposes.
```

Comment:
The apparently intended purpose of this document actually spans both agent and server configuration. The distinctions aren't well specified, and ultimately I'm not sure this really helps much.

Deploying a containerized Pbench Agent isn't an "administrative" function.